src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -2512,8 +2512,10 @@ private CSDiagnosticInfo NotFound(SyntaxNode where, string simpleName, int arity
             // for attributes, suggest both, but not for verbatim name
             if (options.IsAttributeTypeLookup() && !options.IsVerbatimNameAttributeTypeLookup())
             {
-                // just recurse one level, so cheat and OR verbatim name option :)
-                NotFound(where, simpleName, arity, whereText + "Attribute", diagnostics, aliasOpt, qualifierOpt, options | LookupOptions.VerbatimNameAttributeTypeOnly);
+                // Check for generic attribute
```

Comment:
nit: the comment doesn't provide much benefit in terms of clarifying the purpose or intent. It could be removed.